src/azure-cli/azure/cli/command_modules/acr/_params.py

```diff
@@ -169,6 +169,16 @@ def load_arguments(self, _):  # pylint: disable=too-many-statements
         c.argument('read_enabled', help='Indicates whether read operation is allowed.', arg_type=get_three_state_flag())
         c.argument('write_enabled', help='Indicates whether write or delete operation is allowed.', arg_type=get_three_state_flag())
 
+    with self.argument_context('acr artifact-streaming') as c:
+        c.argument('repository', help="The name of the repository.", required=True)
+        c.argument('image', arg_type=image_by_tag_or_digest_type, required=True)
+        c.argument('enable_streaming', help="Indicates whether artifact streaming is enabled for a repository.", required=True, arg_type=get_three_state_flag())
+
+    with self.argument_context('acr artifact-streaming operation') as c:
+        c.argument('repository', help="The name of the repository.", required=False)
+        c.argument('image', arg_type=image_by_tag_or_digest_type, required=False)
+        c.argument('operation_id', options_list=['--id'], required=False, help="The ID returned when creating a streaming artifact.")
```

Comment:
"required=False" can be omitted.
```suggestion
        c.argument('repository', help="The name of the repository.")
        c.argument('image', arg_type=image_by_tag_or_digest_type)
        c.argument('operation_id', options_list=['--id'], help="The ID returned when creating a streaming artifact.")
```